user_guide/data_management_access_permissions_basic.rst

```diff
@@ -0,0 +1,112 @@
+helps to ensure that all the information is safe and available right where it is required, and all the users gain access 
+to the features and capabilities meeting their needs.
+
+This guide is aimed to explain the basic ideas of the access and permission settings. If you are the system 
+administrator, please address the :ref:`Access and Permissions Management <user-guide-user-management-permissions>`
+guide.
+
+What Define the Permissions
+---------------------------
+
+Access and permission rights of each user depend on the following: 
+
+- Organizations and units, to which the user belongs: so users with the same role, working in different offices or 
+  subdivisions may have different access to different information. The list of organizations and units to which a user 
+  belongs is defined by the system administrator in the *"Access Settings"*. 
+  
+- Roles, assigned to the user: for example the sales representatives and system administrators need very different data 
+  and have different access to it. The roles of a user are defined by the system administrator for each user in the 
+  *"Groups and Role"*.
+
+- Ownership type of the entity: when you create a record, you may define its *"Owner"*: these may be the user who can 
+  manage the records or a business unit/organization the members whereof can manage the record. Some record types have 
+  the ownership type *"None"*, which means that all the users can see them. Ownership type of each entity is defined by 
+  the system administrator. 
+
+
+What You Can Be Permitted
+-------------------------
+
+Access to Functionalities
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users with specific roles may be allowed or not allowed to use specific functionality. For example, some users may not 
+have capability to assign tags, to share grids, or to create new users.
+
+These are defined by the system administrator in the *"Capabilities section"* of each role. Only users with a role, for 
+which a capability is enabled, will be able to use the functionality. 
+
+.. note::
+
+    Capabilities of different roles are compounded, i.e. if one role has the right to merge entities, the user will 
+    have this right, regardless of the settings defined for other roles of this user.
+    
+
+Actions with Entities    
+^^^^^^^^^^^^^^^^^^^^^
+
+There is a number of actions that can be permitted for each combination of the role, organization/unit and entity.
+For example, it can be defined that any user with the role "Sales Representative" who belongs to the business unit 
+"California" can create and edit records of Customers, if this records were created by this user or by any other user 
+from this business unit, and can only view (but not edit) similar records created by users from other business units 
+(e.g. "Texas" or "Florida"). At the same time, for users with another role, for example "System Administrator", this 
+records may be even not seen.
```

Comment:
Comment: The second sentence ("For example... "Texas or Florida") is a little long and confusing. I think it would still make its point well enough if it ended at "business unit." In other words: "For example, it's possible to make it so any user with a "Sales Representative" role who belongs to the "California" business unit can create and edit records of Customers, as long as those records have been created by that user or by any other user from their business unit."